xml/System.Windows/Window.xml

```diff
@@ -2362,10 +2362,7 @@
   
  Setting <xref:System.Windows.WindowStartupLocation.CenterScreen> causes a window to be positioned in the center of the screen that contains the mouse cursor.  
   
- Setting <xref:System.Windows.WindowStartupLocation> to <xref:System.Windows.WindowStartupLocation.CenterOwner> causes a window to be positioned in the center of its owner window (see <xref:System.Windows.Window.Owner%2A>), if specified. The owner window can be either another WPF window or a non-WPF window.  
-  
-> [!NOTE]
->  For more information about WPF windows with non-WPF windows, see [WPF and Win32 Interoperation](~/docs/framework/wpf/advanced/wpf-and-win32-interoperation.md) and <xref:System.Windows.Interop.WindowInteropHelper>.  
+ Setting <xref:System.Windows.WindowStartupLocation> to <xref:System.Windows.WindowStartupLocation.CenterOwner> causes a window to be positioned in the center of its owner window (see <xref:System.Windows.Window.Owner%2A>), if specified. The owner window must be a WPF window.  
```

Comment:
Two nits unrelated to your PR, but important to fix:
- The first xref should be \<xref:System.Windows.Window.WindowStartupLocation>. Or better, just, "The \`WindowStartupLocation\` property".
- The second xref should end with "?displayProperty=nameWithType"

Reply:
Thanks! I’m away at the moment, but will get back to this.

Reply:
Done - good spot @rpetrusha! I also change a couple of other cases of the same enum/property xref mix-up which were out of view of the diff.